blog-posts/return-to-office-interview.md

```diff
@@ -0,0 +1,142 @@
+<small>Virtuelle Weihnachtsfeier @ Satellytes. Danke and Georgios and Fabian für die Organisation!</small>
+
+<br />
+
+**Eric:** Klar auf der zwischenmenschlichen Ebene oder innerhalb der Teams hat das weiter sehr gut funktioniert. Die Teams haben schnell gemerkt, dass sie sich die Zeit nehmen müssen und auch dürfen, auch mal über Dinge außerhalb der Arbeit zu plaudern.
+Aber trotzdem habe ich persönlich vieles Vermisst, was vor Corona selbstverständlich war. Man trifft sich zwar immer noch häufig in Zoom-Meetings, aber da hat man immer eine Agenda. Der kurze Small-Talk am Anfang und am Ende ist für mich persönlich kein Ersatz für die längeren Zusammentreffen, die im Büroalltag ständig zustande kommen.
+Vor Corona ist es schon regelmäßig passiert, dass man sich mal an der Kaffeemaschine über den Weg gelaufen ist und sich dann auch mal länger über irgendein Casual-Thema unterhalten hat. Außerdem sind wir auch ständig gemeinsam zum Mittagessen gegangen. Dabei hat man mehr Zeit gefunden über Dinge zu reden, die über die aktuellen Projektthemen hinausgehen. Und genau diese Feinheiten und Hintergründe braucht es meiner Meinung nach, damit ich auf unsere Mitarbeiter richtig eingehen und bei Bedarf helfen kann.
+
+**Die Fallzahlen sinken ja wieder. Es ist absehbar, dass die Home-Office-Pflicht bald ausläuft. Trotzdem überlegen viele, wie sie zukünftig arbeiten wollen. Welche Gründe seht ihr, warum die Leute vom Büro aus arbeiten wollen?**
+
+**Gholam:** Die Motivation dazu ist meiner Meinung nach sehr von den individuellen Vorlieben und Voraussetzungen jedes Einzelnen abhängig. Wir spüren zum Beispiel, dass viele auch einfach Lust darauf haben, ihre Kollegen wieder häufiger zu sehen um unmittelbarer zusammenarbeiten oder auch mal nur gemeinsam Mittag essen gehen zu können.
+Man sieht auch, dass nicht bei allen der Platz oder das Umfeld gegeben ist um entspannt von zu Hause aus arbeiten zu können. Wenn ich in meiner Wohnung wenig Platz habe und jeden Mittag um 12 Uhr die Kinder von der Schule kommen und beschäftigt werden wollen, bin ich wahrscheinlich auch einfach froh darüber einen ruhigen Ort – in dem Fall das Büro – zu haben, von dem ich konzentriert aus arbeiten kann.
+
+**Welche konkreten Vorteile seht ihr darin, wenn wieder mehr Mitarbeiter häufiger gemeinsam vom Büro aus zu arbeiten?**
+
+**Gholam:** Mein Essen am Montag! Das ist ein klarer Vorteil für jeden Mitarbeiter, wenn ich Montags für alle Essen koche (lacht). In Zeiten von Corona musste das leider immer wieder pausieren. Aber das plane ich auf jeden Fall wieder aufzunehmen. sobald das sicher möglich ist.
+
+<br />
+
+![Ja, es gab sie wirklich!](images/return-to-office-interview-lasagna.jpg)
+<small>Ja, es gab sie wirklich!</small>
+<br />
+
+**Eric:** Ja, die Lasagne war Bombe! Dafür habe ich sogar den Schnitt im Daumen in Kauf genommen (lacht). Das war eine tolle Sache.
```

Comment:
🔴 please change 

```suggestion
**Eric:** Ja, die Lasagne war genial! Dafür habe ich sogar den Schnitt im Daumen in Kauf genommen (lacht). Das war eine tolle Sache.
```

That is too much of slang especially with the word "Bombe". Can we please rephrase that with "genial" or similar ? I really dislike the thoughts induced by the word when reading it (it's super okay in a personal chat but not here.)